lib/galaxy/util/string_conversion.py

```diff
@@ -0,0 +1,15 @@
+"""
+Safe conversion to unicode
+"""
+
+
+def to_unicode( a_string ):
+    """
+    Convert a string to unicode in utf-8 format; if string is already unicode,
+    does nothing because string's encoding cannot be determined by introspection.
+    """
+    a_string_type = type( a_string )
+    if a_string_type is str:
+        return unicode( a_string, 'utf-8' )
+    elif a_string_type is unicode:
+        return a_string
```

Comment:
You will get an exception. I copied this function from https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/web/framework/helpers/__init__.py#L101 
I was thinking about adding a check for `None`, but since this is already in use in other places in the code I was thinking to best not change its behaviour.
In fact I think the to_unicode function should be moved out from `lib/galaxy/web/framework/helpers/__init__.py` ... what do you think about that @nsoranzo ?